code/test_planning_top/behavior_agent/src/behavior_agent/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import behavior_tree
+from . import behaviours
```

Comment:
This seems unnecessary

Reply:
this is deleted in https://github.com/ll7/paf22/pull/170/commits/48f63365232b9939d859252c7f5d7560637d498f